packages/flutter/lib/src/material/drawer.dart

```diff
@@ -275,14 +275,21 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
                     child: new Focus(
                       key: _drawerKey,
                       child: config.child
-                    )
-                  )
-                )
-              )
-            ]
-          )
-        )
+                    ),
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ),
       );
     }
   }
+  @override
+  Widget build(BuildContext context) {
+    return new ListTileTheme(
+      style: ListTileStyle.drawer,
+      child: _buildDrawer(context),
```

Comment:
I would inline the _buildDrawer method unless it's called anywhere else, saves a jump and doesn't really make the code any less readable.